src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -134,19 +134,29 @@ public static void main(String[] args) {
 		EventQueue.invokeLater(new Runnable() {
 			public void run() {
 				Application app = new Application("Jobs 2D");
-				DrawerFeature.setupDrawerPlugin(app,app.getFreePanel());
-				CommandsFeature.setupCommandManager();
-				DeviceUsageFeature.setupDeviceUsageManager();
-				DriverFeature.setupDriverPlugin(app);
-				DriverFeature.setUpDriverNameLabelChangeManager();
+
+				DrawerFeature drawerFeature = new DrawerFeature();
+				drawerFeature.setup(app,app.getFreePanel(),DriverFeature.getDriverManager());
+
+				CommandsFeature commandsFeature = new CommandsFeature();
+				commandsFeature.setup(app,app.getFreePanel(),DriverFeature.getDriverManager());
+
+				DeviceUsageFeature deviceUsageFeature = new DeviceUsageFeature();
+				deviceUsageFeature.setup(app,app.getFreePanel(),DriverFeature.getDriverManager());
+
+				DriverFeature driverFeature = new DriverFeature();
+				driverFeature.setup(app, app.getFreePanel(),DriverFeature.getDriverManager());
+				driverFeature.setUpDriverNameLabelChangeManager();
+
+				RecordingFeature recordingFeature = new RecordingFeature();
+				recordingFeature.setup(app, app.getFreePanel(),DriverFeature.getDriverManager());
+
```

Comment:
To make some sense out of the refactoring, we should be able to setup the features automatically without explicit setup calls. Try adding features manager facility for adding the features and configuring them all at once.